src/components/client/one-time-donation/Steps.tsx

```diff
@@ -67,7 +67,7 @@ export default function DonationStepper({ onStepChange }: DonationStepperProps)
     return session && session.accessToken ? true : false
   }
 
-  initialValues.isAnonymous = !isLogged()
+  initialValues.isAnonymous = isLogged()
```

Comment:
this change doesn't look good to me. 
Being logged it is the opposite of being anonymous.

Yes - you are probably right - this line is most likely pointless.
I will take a look and do some tests.
I believe I introduced this bug.

Reply:
Tell me when you are done testing, if it is safe to remove the whole line, so I can amend the commit asap.

Reply:
Yep. Just remove the line. It will be fine.